cmd/srcd/cmd/web.go

```diff
@@ -0,0 +1,133 @@
+// Copyright © 2018 NAME HERE <EMAIL ADDRESS>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/signal"
+	"time"
+
+	"github.com/pkg/browser"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	api "github.com/src-d/engine-cli/api"
+	"github.com/src-d/engine-cli/cmd/srcd/daemon"
+)
+
+var webCmd = &cobra.Command{
+	Use:   "web",
+	Short: "Start web interfaces for source{d} tools",
+}
+
+var webSQLCmd = &cobra.Command{
+	Use:   "sql",
+	Short: "Start gitbase web",
+	Run: func(cmd *cobra.Command, args []string) {
```

Comment:
This function and the one given to `webParseCmd` are basically the same one, consider unifying.